man/systemd-measure.xml

```diff
@@ -104,6 +104,16 @@
 
         <xi:include href="version-info.xml" xpointer="v252"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><command>pcrpkey</command></term>
+
+        <listitem><para>This commands prints the public key either given with <option>--public-key=</option>,
+        or extracted from the certificate given with <option>--certificate=</option> or the private key given
+        with <option>--private-key=</option>.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
```

Comment:
hmm, so systemd-sbsign now has "validatekey" and systemd-measure has "pcrpkey".

Both these operations are in a specific tool but conceptually pretty generic, i.e. key valdation and conversion. Neither have much to do with the tool they are placed in. And in particular, "pcrpkey" has nothing really to do with pcrs, does it?

maybe byte the bullet and move both into a new tool systemd-keymgmt or so? 

and maybe rename "pcrpkey" to just "public"?

Reply:
and maybe rename "validate-key" to just "validate" if it is in its own tool "systemd-keymgmt" which aleady contains the word "key" in its name.